tests/rbd_mirror/test_rbd_mirror_daemon_status.py

```diff
@@ -0,0 +1,54 @@
+from tests.rbd_mirror import rbd_mirror_utils as rbdmirror
```

Comment:
The filename is not concurring with the operations being done in this file. Can we change the filename accordingly?

Reply:
We are unable to change file name since same file "rbd_mirror_utils" we are are using in other module as well.

Reply:
As per the discussion with Manasa and Mohit, we are continuing to keep the file name unchanged.